components/partials/markets/market-row.vue

```diff
@@ -218,6 +284,16 @@ export default Vue.extend({
       const marketRoute = getMarketRoute(market)
 
       return marketRoute || { name: 'index' }
+    },
+
+    isBaycWeth(): boolean {
```

Comment:
When you decouple the component as I've mentioned above, make that component have this switch (a couple of computed properties that check whether some condition is met and then render "Powered by" based on them).

Also, the check should be based on the slug or ticker. I'd prefer slug because we do checks with slugs everywhere